test/infer/test_autoguide.py

```diff
@@ -244,7 +244,7 @@ def model():
     class _AutoGuide(AutoDiagonalNormal):
         def __call__(self, *args, **kwargs):
             return substitute(
-                super(_AutoGuide, self).__call__, {"_auto_latent": x_init}
+                super(_AutoGuide, self).__call__, {"_auto_latent": x_init[None]}
```

Comment:
the old jax is smart here too: this latent value should have shape `(1,)`, rather than `()`.